### PR TITLE
Add ability to query cluster accesses granted

### PR DESCRIPTION
**Pull request by jastBytes**
Currently the API provides the ability to query information about who has access to what in terms of cluster access, but lacks the ability to query the actual binding from which these permissions originate. This PR is about adding such an API.

### CODE REVIEW

File: config/dex/config.yaml [hunk 1]
```diff
@@ -1,9 +1,6 @@
 issuer: http://127.0.0.1:5556
 storage:
-  config:
```
Comment (christianhuening) [inline config/dex/config.yaml:3]: what does this do? were we using a dex sqlite database befor?

File: config/dex/config.yaml [hunk 2]
```diff
@@ -1,9 +1,6 @@
...
-    file: /tmp/dex.db
-    inCluster: true
-  type: sqlite3
+  type: memory
 logger:
   level: debug
```
Comment (christianhuening) [inline config/dex/config.yaml:5]: is log level `debug` as default a good idea? Is it ensured nothing critical is logged?

**Review by christianhuening — APPROVED** [review]
i had two questions about how dex is configured, but nothing critical.

File: config/dex/config.yaml (hunk 1 above)
Comment (jastBytes) [reply config/dex/config.yaml:3]: It's just for the tests. In general we do not use DEX at the moment.

File: config/dex/config.yaml (hunk 2 above)
Comment (jastBytes) [reply config/dex/config.yaml:5]: Same here, just for the testcases, so debugging is fine here.

**Pull request merged**